src/OSPSuite.Presentation/Presenters/Importer/ColumnMappingPresenter.cs

```diff
@@ -526,6 +526,9 @@ private void setDescriptionForRow(ColumnMappingDTO model, bool isColumn)
                   break;
                case ColumnMappingDTO.ColumnType.Mapping:
                   model.Source = new MappingDataFormatParameter(model.ExcelColumn, new Column() {Name = model.MappingName, Unit = new UnitDescription(UnitDescription.InvalidUnit)});
+                  var unit = _format.ExtractUnits(model.ExcelColumn, _columnInfos.First(x => x.DisplayName == model.MappingName).SupportedDimensions);
```

Comment:
Very hard for me to review here. Are you sure here that you have one column with the displayName == MAppingName?

Reply:
I am creating a temporal variable for a cleaner code. It should not be the case that the First throws because we are getting the supported dimensions from a valid mapping column. So, if the MappingName does not exist on the columnInfos collection, it means this is not a valid mapping. I could create a specific error for it although this is not an error for the user but for the developer. An error here means the code is wrong and not anything done by the user.